.github/workflows/generate_library.yml

```diff
@@ -40,7 +42,7 @@ jobs:
       working-directory: ansys-grantami-serverapi-openapi
 
     - name: "Build client library"
-      run: mvn -Dbuild-id=${{ github.run_number }} -s .m2/settings.xml compile
+      run: mvn -Dbuild-id=${{ github.run_number }} ${{ env.IS_RELEASE_BRANCH == 'true' && '-Dis-release' || '' }} -s .m2/settings.xml compile
```

Comment:
I had forgotten that for Github, `env.IS_RELEASE_BRANCH`, even if true, doesn't evaluate to true, and it needs to be compared to `'true'`...

Reply:
Yep, envs are always strings...